plugins/osdn/factory/factory.go

```diff
@@ -24,3 +24,10 @@ func NewPlugin(pluginType string, osClient *osclient.Client, kClient *kclient.Cl
 
 	return nil, nil, nil
 }
+
+func IsMultitenantNetworkPlugin(pluginType string) bool {
```

Comment:
Enabling NetNamespace REST resource for ovs-subnet is not an issue but the VNID repair controller runs every 15 mins and makes a bunch of etcd calls and these will return nothing in case of ovs-subnet. So enabling NetNamespace/VNID only when it is necessary will save some cpu/io cycles.